test/JDBC/input/BABEL-1994-CHAR.sql

```diff
@@ -12,13 +12,13 @@ create index pad_c5nn_idx on pad(c5nn);
 go
 
 select
-    concat('[', c5nn, ']') as c5nn_concat,
+    pg_catalog.concat('[', c5nn, ']') as c5nn_concat,
     '[' + c5nn +']' as c5nn_to_text_concat,
-    concat('[', cast(c5nn as varchar), ']') as c5nn_to_varchar_concat,
-    concat('[', c5n, ']') as c5n_concat,
+    pg_catalog.concat('[', cast(c5nn as varchar), ']') as c5nn_to_varchar_concat,
+    pg_catalog.concat('[', c5n, ']') as c5n_concat,
     '[' + c5n + ']' as c5n_to_text_concat,
-    concat('[', cast(c5n as varchar), ']') as c5n_to_varchar_concat,
-    concat('[', cast(c5n as pg_catalog.name), ']') as c5n_to_name_concat
+    pg_catalog.concat('[', cast(c5n as varchar), ']') as c5n_to_varchar_concat,
+    pg_catalog.concat('[', cast(c5n as pg_catalog.name), ']') as c5n_to_name_concat
```

Comment:
`name` datatype is not castable to varchar so it will throw an error with new definitions, so appended PG_CATALOG in that to preserve the test, although this is not required at other places, so removed that.